Tools/Scripts/webkitdirs.pm

```diff
@@ -2946,6 +2950,27 @@ sub cmakeArgsFromFeatures(\@;$)
     return @args;
 }
 
+sub vcpkgArgsFromFeatures(\@;$)
```

Comment:
vcpkgArgsFromFeatures is useless. It just returns a constant string.
I think we don't have to add avif and skia now. It's easy to add them later.

Reply:
I haven't gotten around to looking for feature flags yet so this is stubbed out.

Reply:
Then, let's remove it for now.